project/plugins.sbt

```diff
@@ -3,3 +3,5 @@ addSbtPlugin("org.scala-sbt" % "sbt-contraband"  % "0.4.1")
 addSbtPlugin("com.lightbend" % "sbt-whitesource" % "0.1.9")
 
 scalacOptions += "-language:postfixOps"
+
+libraryDependencies += "org.eclipse.jgit" % "org.eclipse.jgit.pgm" % "3.2.0.201312181205-r"
```

Comment:
sbt-houserules already contains sbt-git.

Reply:
ok, even if I'm not sure to inject in the sbt classpath a specific version of jgit (general observation)